Packs/DeveloperTools/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,3 @@
+
+#### Developer Tools
```

Comment:
```suggestion
### Developer Tools
```

Reply:
it failed on me for not having a first level header for this